weave/trace/op_extensions/error_once.py

```diff
@@ -0,0 +1,12 @@
+from typing import Callable
+
+LOG_ONCE_MESSAGE_SUFFIX = " (subsequent messages of this type will be suppressed)"
+logged_messages = []
+
+
+def log_once(log_method: Callable[[str], None], message: str) -> None:
```

Comment:
a bit confusing wrt the file name `error_once`. Also a docstring here would be nice (with the cannonical example)

Reply:
this is to be used mainly in generators? for loops?